README.md

```diff
@@ -64,6 +64,7 @@ Set values in the json payload via the `options` key:
   "g": true,
   "c": true,
   "t": 4,
-  "x": 5
+  "x": 5,
+  "l": 10
```

Comment:
This should be quickly described before the example like the other options (see comment right above the code snippet).

Reply:
done